packages/application-shell/src/components/navbar/navbar.mod.css

```diff
@@ -329,14 +334,21 @@
   display: -webkit-box;
   -webkit-line-clamp: 2;
   -webkit-box-orient: vertical;
-  overflow: hidden;
   color: var(--color-solid);
   /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
   font-weight: var(--font-weight-for-navbar-link-when-hovered);
   text-decoration: none;
   flex: 1;
   padding: var(--spacing-25) var(--spacing-25) var(--spacing-25)
     var(--spacing-30);
+  transition: padding 150ms ease-out;
+}
+
+.sublist-item:not(.sublist-item__active):hover .text-link-sublist,
+.sublist-item:not(.sublist-item__active):focus-within .text-link-sublist {
+  /* additional left padding on hover and focus */
+  padding: var(--spacing-25) var(--spacing-25) var(--spacing-25)
+    calc(var(--spacing-30) + var(--spacing-20));
```

Comment:
I've revised the strategy in the last update. It appeared that the `'overflow: hidden'`s I removed might not be crucial (I couldn't detect differences, but reviewers may catch any regressions). As a result, I've shifted to adding the focus ring and extra padding directly to the `<a>` (`.text-link-sublist`) element instead of its parent.

This modification ensures compatibility across all browsers without the need for any workarounds.

For now, I'm holding off on the scheduled sync as there are no pending questions on my end.

Your feedback is most welcome!